frontend/src/components/placementVisuals/Placement.tsx

```diff
@@ -188,7 +154,7 @@ function Placement({ stepID, sampleIDs }: PlacementProps) {
                         </Col>
                         {usesSamplesheet && <Col span={3} >
                             <Button
-                            onClick={handleGetSamplesheet}
+                                onClick={handleGetSamplesheet}
                             // disabled={!isPlacementComplete}
```

Comment:
With PacBio coming with smrt cells that can be empty i don't this this line is making a come back. You can likely remove the commented line for good.